usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -374,6 +392,59 @@ def query_elasticsearch(self) -> list:
 
         return response
 
+    def query_elasticsearch_subawards(self) -> list:
```

Comment:
This method returns a `Response`
```suggestion
    def query_elasticsearch_subawards(self) -> Response:
```